src/core/data-tests.js

```diff
@@ -23,11 +23,34 @@ export const name = "core/data-tests";
 const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
-  return hyperHTML.bind(document.createElement("li"))`
+  
+  let modifiedHref = href.split("/").pop().split(".html")[0];
+  
+  let emojiList = document.createElement('i');
+  
+  let requiresConnectionEmoji = hyperHTML.bind(document.createElement("i"))` 🔒 `;
```

Comment:
These elements should only be created inside the if statements below. Otherwise, they are getting created unnecessarily if not needed (and then thrown away).